cumulus_library_opioid/manifest.toml

```diff
@@ -7,44 +7,44 @@ file_names = [
     "vocab/additional_rules_builder.py",
 ]
 
-[sql_config]
-file_names = [
-    "define_dx.sql",
-    "define_dx_sepsis.sql",
-    "define_dx_sud.sql",
-    "define_lab.sql",
-    "define_rx.sql",
-    "define_rx_buprenorphine.sql",
-    "define_rx_naloxone.sql",
-    "define_rx_opioid.sql",
-    "table_study_period.sql",
-    "table_dx.sql",
-    "table_dx_sepsis.sql",
-    "table_lab.sql",
-    "table_rx.sql",
-    "version.sql"
-]
+# [sql_config]
+# file_names = [
+#     "define_dx.sql",
+#     "define_dx_sepsis.sql",
+#     "define_dx_sud.sql",
+#     "define_lab.sql",
+#     "define_rx.sql",
+#     "define_rx_buprenorphine.sql",
+#     "define_rx_naloxone.sql",
+#     "define_rx_opioid.sql",
+#     "table_study_period.sql",
+#     "table_dx.sql",
+#     "table_dx_sepsis.sql",
+#     "table_lab.sql",
+#     "table_rx.sql",
+#     "version.sql"
+# ]
```

Comment:
i've been doing this manually each time im working and then undoing it on commit - since we now remember that the opioid study is not working in library v2.3, im inclined to leave it commented until we migrate these tables